### PR TITLE
created reusable components and added icon library

### PR DESCRIPTION
**Pull request by GabriellaMSanchez**
Created reusable components for the Main Navigation Page, Search Bar, and Navigation Buttons. I also imported an icon library from FontAwesome to serve as the icons for the Navigation Buttons.

### CODE REVIEW

File: src/SearchBar.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        type="text" 
+        placeholder="Search a Park"
+        style={{border: '3px solid black'}}
+        autoFocus
+        spellCheck
+    />
+    )
+}
+
+export default SearchBar;
```
Comment (DanielRyanSmith) [inline src/SearchBar.js:14]: New line at end of file

**Review by DanielRyanSmith — COMMENTED** [review]
It would be useful to have more descriptive PR descriptions and screenshots of changes 🙁

File: src/SearchBar.js (hunk 1 above)
Comment (GabriellaMSanchez) [reply src/SearchBar.js:14]: Fixed!

**Comment by GabriellaMSanchez** [comment]
Screenshot of how the Main Nav Page looks with changes

File: src/SearchBar.js (hunk 1 above)
Comment (DanielRyanSmith) [reply src/SearchBar.js:14]: It looks like there's still not a new line at the end here 😅

**Review by DanielRyanSmith — APPROVED** [review]
LGTM after adding the new line

File: src/SearchBar.js (hunk 1 above)
Comment (GabriellaMSanchez) [reply src/SearchBar.js:14]: Screen Shot 2023-10-02 at 11 31 24 AM

Comment (GabriellaMSanchez) [reply src/SearchBar.js:14]: Idk why its not showing! Its been saved like this 😓

Comment (DanielRyanSmith) [reply src/SearchBar.js]: I can see it now!

**Comment by GabriellaMSanchez** [comment]
Just re-pushed so the NL is there. It was there when I went back to the branch, but lmk if it is still not appearing

**Pull request merged**